srcpkgs/kdenetwork-filesharing/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'kdenetwork-filesharing'
+pkgname=kdenetwork-filesharing
+version=21.04.3
+revision=1
+build_style=cmake
+hostmakedepends="extra-cmake-modules qt5-qmake qt5-host-tools kcoreaddons
+ $(vopt_if samba 'samba') gettext ki18n kauth"
+makedepends="kconfig-devel kio-devel kdeclarative-devel
+ $(vopt_if samba 'samba-devel')"
+depends="dolphin $(vopt_if samba 'samba')"
```

Comment:
No need to quote the packages inside the `vopt_if` blocks.

Reply:
Sorry but I'd prefer leave it there because improves the readability distinguishing well the build condition from the package/s which will be installed mostly when the build condition and package name are equal.